packages/shared/components/modals/CollectibleActionsModal.svelte

```diff
@@ -10,13 +10,39 @@
     import { activeProfile, updateActiveProfile } from '@core/profile/stores'
     import { Icon as IconEnum } from '@auxiliary/icon'
 
-    export let modal: Modal = undefined
+    export let modal: Modal
     export let nft: INft
 
     $: url = nft?.parsedMetadata?.uri && composeUrl(nft.parsedMetadata.uri)
     $: isLocked = nft.timelockTime > $time.getTime()
     $: isCurrentPfp = $activeProfile.pfp?.id === nft.id
 
+    $: MENU_ITEMS = [
+        {
+            icon: IconEnum.Receive,
+            title: localize('views.collectibles.details.menu.download'),
+            disabled: true,
+            onClick: (): void => {},
```

Comment:
is this `onClick` necessary?

Reply:
it is. MenuItem right now requires a function for the onclick. It's not optional